.NET/Microsoft.Recognizers.Text.Number/Spanish/Parsers/SpanishNumberParserConfiguration.cs

```diff
@@ -95,6 +97,20 @@ public override IEnumerable<string> NormalizeTokenSet(IEnumerable<string> tokens
                 result.Add(token);
             }
 
+            // The following piece of code is needed to compute the fraction pattern number+'y medio'
+            // e.g. 'cinco y medio' ('five and a half') where the numerator is omitted in Spanish.
+            // It works by inserting the numerator 'un' ('a') in the list fracWords
+            // so that the pattern is correctly processed.
+            var resLen = result.Count;
+            if (resLen > 2)
+            {
+                if (result[resLen - 1] == "medio" && result[resLen - 2] == "y")

```

Comment:
Please move any hard coded terms to YAML resources.

Reply:
moved